mlflow/projects/databricks.py

```diff
@@ -229,8 +229,8 @@ def run_databricks(uri, entry_point, version, parameters, experiment_id, cluster
                                  "on Databricks.")
 
     # Fetch the project into work_dir & validate parameters
-    work_dir = _fetch_project(uri=uri, use_temp_cwd=True, version=version,
-                              git_username=git_username, git_password=git_password)
+    work_dir = _fetch_project(uri=uri, version=version, git_username=git_username,
```

Comment:
btw, out of curiosity, do we really need `git_username` and `git_password` everywhere? It seems unlikely anyone would pass these as arguments to `mlflow`, but they are threaded everywhere.

Reply:
Good question, we have them to enable running git projects in environments where Git credentials aren't implicitly available (e.g. in Databricks notebooks)